### PR TITLE
[BD-05] [TNL-7310] [BB-3388] Pluggable ORA Submission Editor - TinyMCE

### PR DESCRIPTION
**Pull request by shimulch**
Depends on https://github.com/edx/edx-ora2/pull/1574

This PR adds TinyMCE v5 as the WYSIWYG editor for ORA Response.

**JIRA tickets**:
- https://openedx.atlassian.net/browse/TNL-7310
- https://tasks.opencraft.com/browse/BB-3388

**Discussions**: None

**Dependencies**: https://github.com/edx/edx-ora2/pull/1574

**Screenshots**: 
Screenshot from 2021-01-22 10-34-42

**Sandbox URL**: https://bb3388.sandbox.opencraft.hosting/

**Merge deadline**: None

**Testing instructions**:

1. Clone this PR on your local devstack. Install using make install-local-ora.
2. Create an ORA block. On the settings tab select ``WYSIWYG Editor`` from Response Editor Dropdown.
3. Publish ORA Block
4. Check that TinyMCE editor loaded in both LMS & Studio. 
5. Write on the editor, check ``Save Your Progress`` becomes enabled.
6. Click on ``Save Your Progress`` and reload the page. You should come back where you left off.
7. Check if the submission process working.

**Author notes and concerns**:

1. This PR includes TinyMCE v5 and all of the assets related to the editor.

**Reviewers**
- [ ] @viadanna
- [ ] @xitij2000
- [ ] @giovannicimolin
- [ ] edX reviewer[s] TBD

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @shimulch! I've created [BLENDED-732](https://openedx.atlassian.net/browse/BLENDED-732) to keep track of it in Jira.   More details are on the [BD-05](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1529970851/BD-05+ORA+v3+Usability+Moderation) project page.


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.

File: openassessment/xblock/studio_mixin.py [hunk 1]
```diff
@@ -151,8 +157,10 @@ def editor_context(self):
             'feedbackprompt': self.rubric_feedback_prompt,
             'feedback_default_text': feedback_default_text,
             'text_response': self.text_response if self.text_response else '',
+            'text_response_editor': self.text_response_editor if self.text_response_editor else '',
```
Comment (viadanna) [inline openassessment/xblock/studio_mixin.py:160]: Would it make sense to use the default `text` as [here](https://github.com/edx/edx-ora2/pull/1576/files#diff-76ce3a8b3d4643ee71466f0dc9d6628478b6e76d8c8af079dab15f12cb96b593R882)?

**Review by viadanna — APPROVED** [review]
Good to go 👍

- [x] I tested this on my local devstack.
- [x] I read through the code.

File: openassessment/xblock/studio_mixin.py (hunk 1 above)
Comment (shimulch) [reply openassessment/xblock/studio_mixin.py]: Yes. Updated. Thanks for noticing.

**Comment by giovannicimolin** [comment]
@jnlapierre @mattcarter This is ready for review.

**Review by jnlapierre — APPROVED** [review]
LGTM, please make sure all conflicts are resolved and build checks pass.

**Comment by shimulch** [comment]
@jnlapierre Thanks a lot for review. I've fixed merged issues. I will rebase again once https://github.com/edx/edx-ora2/pull/1574 gets merged. cc @giovannicimolin

**Pull request merged**